### PR TITLE
components/confirm: improved centering

### PR DESCRIPTION
**Pull request by benma**
Fix centering algorithm - before it would center the text in the whole
screen. Now it is centering it in the body part, which is the full
screen without the title bar. This leads to more natural looking
positioning.

### CODE REVIEW

**Comment by benma** [comment]
This puts e.g. the address when confirming an address a bit lower right now.

When we increase the font size in the title and body of confirm dialogs, this will still work nicely. The original PR (https://github.com/digitalbitbox/bitbox02-firmware/pull/416/) was doing more manual work to fix the confirm dialog with bigger sizes, such as manually estimating offsets for titles with and without newlines:

https://github.com/digitalbitbox/bitbox02-firmware/pull/416/files#diff-e2850f41a2f354d9355a3c940009f93aR128-R139

and using custom offsets, reducing font size in some places, etc.

File: src/ui/components/confirm.c [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 #include <hardfault.h>
 #include <screen.h>
+#include <ui/fonts/arial_fonts.h>
```
Comment (x1ddos) [inline src/ui/components/confirm.c:23]: Why is this include needed only now. What happens if you don't include?

Comment (benma) [reply src/ui/components/confirm.c:23]: oops that's a leftover from when i used a font inside _render(). will remove again

**Comment by benma** [comment]
Added a commit to use the empty container (recently added) instead of basically replicating it.

**Pull request merged**